openlibrary/templates/account/view.html

```diff
@@ -52,7 +52,7 @@ <h1 class="details-title">$header_title</h1>
               $ share_links = ctx.get('share_links')
               $:macros.ShareModal(share_links, page_url, share_content, "share-modal-wrapper", show_embed=False)
         </div>
-      $if page:
+      $if page and mb.is_my_page:
         $:macros.databarView(page)
```

Comment:
```suggestion
        $:macros.databarView(page, edit=mb.is_my_page)
```
Only show the edit button to the list's owner.